mdx/guides/rfqt-in-the-0x-api.mdx

```diff
@@ -0,0 +1,164 @@
+
+Furthermore, the request sent from the 0x API to an RFQ-T market maker also includes the API key associated with the trusted integration client. In the same way that makers can blacklist taker addresses, they also have the option to blacklist specific applications that integrate with the 0x API.
+
+In order to retain access to RFQ-T liquidity, it's important for clients of the 0x API to be good citizens of the marketplace. For end users of trusted integrations, being a good citizen means actually filling the orders they claim to intend to fill. And for trusted integrations, it means properly managing their users' intentions; more specifically, it means adhering to the flow of using indicative quotes by default, and firm quotes only when appropriate.
+
+# Quote Types: Indicative vs Firm
+
+In order to improve capital efficiency, the RFQ-T model introduces a new request/response flow: first request pricing, then request a fillable quote.
+
+When a market maker provides a fillable quote, which can be settled by the taker's submitting it to the 0x protocol, the taker has an expectation that the swap will actually settle, that it won't revert due to a lack of available maker assets. To uphold this expectation, makers will refrain from using those same assets as the basis for any other quotes, until the quote's expiry.
+
+However, if every price inquiry required the maker to reserve capital, they would quickly end up with all of their capital in reserve. They would then be unable to provide any other quotes until those outstanding quotes were either filled or expired.
+
+To alleviate this potential capital inefficiency, the RFQ-T model introduces the distinction between "indicative pricing" and "firm quotes." In service of this distinction, the 0x API offers two different resources for the use of RFQ-T.
+
+## Indicative Pricing
+
+The indicative pricing resource is hosted at [`/swap/v0/price`](/docs/api#get-swapv0price) and responds with pricing information, but that response does not contain a full 0x order, so it does not constitute a legitimate transaction that can be submitted to the Ethereum network. Therefore, when a market maker services an indicative request they do not need to reserve any capital for it, leaving their capital available for other quotes to other clients.
+
+## Firm Quotes
+
+The firm quote resource is hosted at [`/swap/v0/quote`](/docs/api#get-swapv0quote) and responds with a full 0x order, which can be submitted to an Ethereum node by the client. Therefore it is expected that the maker has reserved the maker assets required to settle the trade, leaving the order unlikely to revert.
```

Comment:
Referring to "full 0x order" seems a bit confusing as /quote contains calldata that is made up of potentially many orders. Perhaps we say something like "responds with on-chain fillable calldata"